app/main.py

```diff
@@ -14,4 +14,12 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    phrase_lower = phrase.lower()
```

Comment:
You are converting the whole `phrase` to lowercase. This could be inefficient for large strings. Consider converting `letter` to lowercase outside the loop, and then compare characters in their original case.